pymess/models/common.py

```diff
@@ -18,3 +29,105 @@ def create_from_related_objects(self, *related_objects):
             self.create_from_related_object(related_object)
             for related_object in related_objects
         ]
+
+
+class BaseMessage(SmartModel):
+
+    STATE = ChoicesNumEnum()
+
+    sent_at = models.DateTimeField(verbose_name=_l('sent at'), null=True, blank=True, editable=False)
```

Comment:
raději _l v knihovnách. Pokud nepřidáš chamber do instaled_apps tak to nebude fungovat. Proto to raději v knihovnách dělám standardní cestou _( je ugettext_lazy a ugettext nezkracujeme

Reply:
mohlo by to mást případné programátory, kteří toto využijí